pkg/objectnames/objectnames.go

```diff
@@ -30,3 +33,7 @@ func GetMachineConfigName(instanceName, mcpName string) string {
 func GetComponentName(instanceName, mcpName string) string {
 	return fmt.Sprintf("%s-%s", instanceName, mcpName)
 }
+
+func ExtractPoolNameFromRTEDaemonset(dsName, instanceName string) string {
+	return strings.Replace(dsName, fmt.Sprintf("%s-", instanceName), "", 1)
```

Comment:
`strings.Cut` ? https://pkg.go.dev/strings#Cut

Reply:
no longer needed,but thanks for the info